src/plugins/mer/merdeploysteps.cpp

```diff
@@ -689,8 +689,8 @@ void MerMb2RpmBuildStep::stdOutput(const QString &line)
     if (rexp.indexIn(line) != -1) {
         QString file = rexp.cap(1);
         //TODO First replace shared home and then shared src (error prone!)
-        file.replace(QRegExp(QString("^") + Constants::MER_SDK_SHARED_HOME_MOUNT_POINT), m_sharedHome);
-        file.replace(QRegExp(QString("^") + Constants::MER_SDK_SHARED_SRC_MOUNT_POINT), m_sharedSrc);
+        file.replace(QRegExp(QString("^") + Sfdk::Constants::BUILD_ENGINE_SHARED_HOME_MOUNT_POINT), m_sharedHome);
+        file.replace(QRegExp(QString("^") + Sfdk::Constants::BUILD_ENGINE_SHARED_SRC_MOUNT_POINT), m_sharedSrc);
```

Comment:
I would extract these into a separate commit "Mer: Move constants for shared paths to Sfdk". That way it will also become more clear that all the enablers for docker build engine exist on sfdk lib side only.

Reply:
moved to a separate commit